src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Machine> get() {
```

Comment:
Do not use raw types. Always parametrize interfaces and classes if it is possible. The 'get' method should return 'List<Bulldozer>' instead of 'List<Machine>'.